CHANGELOG.md

```diff
@@ -3712,7 +3712,7 @@ Replaced by 3.33.1 during release process.
 
 ### Breaking Changes
 
-Please note, the following 2 breaking changes were included in our [3.0 changlog](https://www.pulumi.com/docs/get-started/install/migrating-3.0/#updated-cli-behavior-in-pulumi-30)
```

Comment:
It looks like this is the only line the changed. But there are tons of changes in the file. Could you remove all the other (whitespace?) changes in this file so only this one line shows up as a change?

Reply:
yah, no clue what i was doing; removed the changes to this file